apps/io-wallet-user-func/src/infra/http/handlers/create-wallet-instance.ts

```diff
@@ -79,6 +81,18 @@ export const CreateWalletInstanceHandler = H.of((req: H.HttpRequest) =>
                 "NEW_WALLET_INSTANCE_CREATED",
               ),
             ),
+            RTE.chainW(() =>
+              sendEmail({
+                html: WalletInstanceActivationEmailTemplate(
+                  "to do", // to do - set the faq link
+                  "to do", // to do - set the correct access link
+                ),
+                subject:
+                  "IT Wallet - Aggiungi i tuoi documenti al Portafoglio di IO",
+                text: "IT Wallet - Aggiungi i tuoi documenti al Portafoglio di IO",
+                to: "test@test.test", // to do - get the email by the fiscal code
```

Comment:
create an abstraction module for this

Reply:
What do you mean by "abstraction module"?

Reply:
you could think of creating a function that given a CF returns the email address and use it here. Then you go to insert the todo only in this function that for now becomes a mock.